src/app_rpv.cpp

```diff
@@ -1702,16 +1684,16 @@ void AppRpv::stress_dislocation(int n)
 
       for(i = 0; i < 3; i ++) {
         for(j = 0; j < 3; j ++) {
-          //stres[i][j] = sigma[i][j]/d; 
+          // stres[i][j] = sigma[i][j]/d; 
           stres[i][j] = sigma[i][j]*d/(d*d + dcore*dcore); //stress converge at core distance 
```

Comment:
stres is a 2nd tensor, and stress is a vector, as used in the code. Sort of
tensor mechanics and solid mechanics. The stress field around a dislocation
is calculated using tensor mechanics, and recorded at each lattice point as
in solid mechanics.
Not sure what I did for the first one, maybe adding a space to make it
aligned better?

On Wed, Jun 3, 2015 at 5:23 PM, Daniel Schwen <notifications@github.com>
wrote:

> In src/app_rpv.cpp
> <https://github.com/idaholab/SPPARKS/pull/3#discussion_r31680996>:
>
> > @@ -1702,16 +1684,16 @@ void AppRpv::stress_dislocation(int n)
> >
> >        for(i = 0; i < 3; i ++) {
> >          for(j = 0; j < 3; j ++) {
> > -          //stres[i][j] = sigma[i][j]/d;
> > +          // stres[i][j] = sigma[i][j]/d;
> >            stres[i][j] = sigma[i][j]*d/(d*d + dcore*dcore); //stress converge at core distance
>
> So what is the difference between stres and stress?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/SPPARKS/pull/3/files#r31680996>.
>



-- 

*********************************************

Yongfeng Zhang, PhD

Staff Scientist

Fuels modeling and simulations

Idaho National Laboratory-MS3840

Idaho Falls, ID 83401

Email: yongfeng.zhang@inl.gov

Phone: 208-5260430